docs/application_config.md

```diff
@@ -58,18 +58,19 @@ This configuration drives resource specific aspects of the application's user in
 All resource configurations must match or extend the `BaseResourceConfig`. This configuration supports the following options:
 1. `displayName`: The name displayed throughout the application to refer to this resource type.
 2. `filterCategories`: An optional `FilterConfig` object. When set for a given resource, that resource will display filter options in the search page UI.
+3. `supportedSources`: An optional `SourcesConfig` object.
 
```

Comment:
Update "Index Users" section in this file to generally cover the configurations for indexing other resources